mne/viz/topomap.py

```diff
@@ -653,6 +668,9 @@ def plot_topomap(data, pos, vmin=None, vmax=None, cmap=None, sensors=True,
 
         .. versionadded:: 0.18
     %(topomap_sphere)s
+    border : float | 'mean'
+        Value to extrapolate to on the topomap borders. If ``'mean'`` then
+        each extrapolated point has the average value of its neighbours.
```

Comment:
```
%(border)s
```
Then add an entry to `mne/utils/docs.py`, including a `.. versionadded:: 0.20` in it.

Reply:
yes, I was going to do that :)